.github/workflows/release.yml

```
@@ -68,3 +68,15 @@ jobs:
           JRELEASER_MAVENCENTRAL_TOKEN: ${{ secrets.JRELEASER_MAVENCENTRAL_TOKEN }}
           JRELEASER_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: ./gradlew jreleaserDeploy
+
+  generate-llms:
+    name: Generate llms.txt after release
+    needs: release
+    # Do not fail the whole release workflow if llms generation has issues.
+    continue-on-error: true
```

actionlint (1.7.8)

when a reusable workflow is called with "uses", "continue-on-error" is not available. only following keys are allowed: "name", "uses", "with", "secrets", "needs", "if", and "permissions" in job "generate-llms"

(syntax-check)